test_get_count.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3
```

Comment:
is this file needed?

Reply:
yes it plays in the roll of testing file ensures that the console.py module and the tests/test_console.py module have appropriate docstrings and adhere to the PEP 8 style guidelines.